tests/test_gpu_examples.py

```diff
@@ -16,9 +16,11 @@
 import os
 import tempfile
 import unittest
+from copy import deepcopy
 from dataclasses import dataclass
 from typing import Any, Dict, List, Union
 
+import bitsandbytes as bnb
```

Comment:
Let's make the bitsandbytes import local (inside the test function), as it is not installed on our CPU CI runners.